modules/home-environment.nix

```diff
@@ -513,6 +513,7 @@ in
     programs.bash.shellAliases = cfg.shellAliases;
     programs.zsh.shellAliases = cfg.shellAliases;
     programs.fish.shellAliases = cfg.shellAliases;
+    programs.nushell.shellAliases = cfg.shellAliases;
```

Comment:
It seems the `home.shellAliases` aren't always compatible with those from nushell.

Would it be possible to remove this and keep `programs.nushell.shellAliases`?

Reply:
Yeah, I think that'd probably make the most sense! I still didnt find a nice way of wrapping the `cfg.shellAliases` to be generally compatible. I'll make that change, and push up a commit, and ping the reviewers